src/gridfs.rs

```diff
@@ -52,12 +52,39 @@ pub struct FilesCollectionDocument {
     pub chunk_size: u32,
     pub upload_date: DateTime,
     pub filename: Option<String>,
-    #[serde(skip_serializing_if = "Option::is_none")]
     pub metadata: Option<Document>,
 }
 
+impl FilesCollectionDocument {
+    /// Returns the total number of chunks expected to be in the file.
+    fn n(&self) -> u32 {
+        Self::n_from_vals(self.length, self.chunk_size)
+    }
+
+    fn n_from_vals(length: u64, chunk_size: u32) -> u32 {
+        let chunk_size = chunk_size as u64;
+        let n = length / chunk_size + u64::from(length % chunk_size != 0);
+        n as u32
```

Comment:
This number is much easier to use in calculations against chunks' `n` fields when it's a `u32`, and needed to be cast to `u32` in a few places anyway. A file would need to be ~1M GB for its total number of chunks to exceed `u32::MAX` with the default chunk size, so it's extremely unlikely that this cast will ever fail.